assets/stylesheets/home.scss

```diff
@@ -3,29 +3,53 @@
 	.cards-container{
 		width: 100%;
 		padding: 20px;
+		box-sizing: border-box;
+		overflow: hidden;
+
 	}
 	.grid-sizer{
 		display: none;
 	}
-	.grid-sizer, .card{
-		width: 230px;
+	.card-wrapper{
+		height: 385px;
+		width: 300px;
+		display: inline-block;
+		position: relative;
+	}
+	.card-wrapper{
 		background-color: #C9BFDE;
 		float: left;
 		margin-bottom: 10px;
 		border-radius: 2px;
-		padding: 3px;
+		padding: 6px;
 		box-sizing: border-box;
 		text-align: center;
 		box-shadow: 0 2px 10px rgba(0, 0, 0, 0.17);
+		transition: all 0.7s;
+		cursor: pointer;
+		text-decoration: none;
+		margin: 5px;
 		.title{
 			margin: 5px;
 			font-size: 20px;
 			font-weight: bold;
 			color: black;
+			max-height: 92px;
 		}
 		.details{
 			text-align: left;
-			margin: 5px;
+			position: absolute;
+			box-sizing: border-box;
+			padding: 10px 3px 3px 3px;
+			width: 97%;
+			bottom: 3px;
+			background: #C9BFDE;
+			height: 35px;
+			box-shadow: 0px -6px 19px #C9BFDE;
+			font-weight: bold;
+		}
+		.preview{
+			color: #6B6B6B;
```

Comment:
I think we should start using our color palet and assign each color thats been used in the basic file.